public/models/game/game.js

```diff
@@ -59,6 +59,18 @@ var Game = Map.extend({
 				newVal.__listSet = {gameId: this.attr("id")};
 				return newVal;
 			}
+		},
+		videoUrl: {
+			serialize: true,
+			get: function (setVal) {
```

Comment:
This is a must fix at some point.  So good to change now.  Basically the same logic in `set` is all that's needed.